{"name": "Uploading file from local drive", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n            failed while waiting 4 seconds\n            to assert Visible\n            for first_by('xpath', '//button[text() = \"Upload\"]')\n\n            reason: NoSuchElementException: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//button[text() = \"Upload\"]\"}\n  (Session info: chrome=68.0.3440.106)\n  (Driver info: chromedriver=2.41.578737 (49da6702b16031c40d63e5618de03a32ff6c197e),platform=Windows NT 10.0.16299 x86_64)\n            screenshot: file://C:\\Users\\ssoloshchenko\\.selene\\screenshots\\1535962116742\\screen_1535962116743.png", "trace": "..\\..\\.virtualenvs\\job_control-U42CF0cz\\lib\\site-packages\\selene\\wait.py:13: in wait_for\n    return condition.fn(entity)\n..\\..\\.virtualenvs\\job_control-U42CF0cz\\lib\\site-packages\\selene\\conditions.py:141: in fn\n    return self.match(element.get_actual_webelement())\n..\\..\\.virtualenvs\\job_control-U42CF0cz\\lib\\site-packages\\selene\\elements.py:205: in get_actual_webelement\n    return self.__delegate__\n..\\..\\.virtualenvs\\job_control-U42CF0cz\\lib\\site-packages\\selene\\elements.py:200: in __delegate__\n    return self._locator.find()\n..\\..\\.virtualenvs\\job_control-U42CF0cz\\lib\\site-packages\\selene\\elements.py:45: in find\n    return self._search_context.find_element(*self._by)\n..\\..\\.virtualenvs\\job_control-U42CF0cz\\lib\\site-packages\\selene\\driver.py:89: in find_element\n    return self._webdriver.find_element(by, value)\n..\\..\\.virtualenvs\\job_control-U42CF0cz\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py:966: in find_element\n    'value': value})['value']\n..\\..\\.virtualenvs\\job_control-U42CF0cz\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py:320: in execute\n    self.error_handler.check_response(response)\n..\\..\\.virtualenvs\\job_control-U42CF0cz\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py:242: in check_response\n    raise exception_class(message, screen, stacktrace)\nE   selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//button[text() = \"Upload\"]\"}\nE     (Session info: chrome=68.0.3440.106)\nE     (Driver info: chromedriver=2.41.578737 (49da6702b16031c40d63e5618de03a32ff6c197e),platform=Windows NT 10.0.16299 x86_64)\n\nDuring handling of the above exception, another exception occurred:\n..\\..\\.virtualenvs\\job_control-U42CF0cz\\lib\\site-packages\\selene\\elements.py:188: in _wait_with_screenshot\n    return wait_for(entity, condition, timeout, polling)\n..\\..\\.virtualenvs\\job_control-U42CF0cz\\lib\\site-packages\\selene\\wait.py:35: in wait_for\n    reason=reason_string), screen, stacktrace)\nE   selenium.common.exceptions.TimeoutException: Message: \nE               failed while waiting 4 seconds\nE               to assert Visible\nE               for first_by('xpath', '//button[text() = \"Upload\"]')\nE   \nE               reason: NoSuchElementException: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//button[text() = \"Upload\"]\"}\nE     (Session info: chrome=68.0.3440.106)\nE     (Driver info: chromedriver=2.41.578737 (49da6702b16031c40d63e5618de03a32ff6c197e),platform=Windows NT 10.0.16299 x86_64)\n\nDuring handling of the above exception, another exception occurred:\ntest_login.py:45: in test_upload\n    MainPage().test_upload_from_pc().jobs_list.should(have.texts('vector .pdf', 'giant', 'chine', 'destro'))\n..\\src\\main_page.py:56: in test_upload_from_pc\n    browser.element(by.xpath('//button[text() = \"Upload\"]')).click()\n..\\..\\.virtualenvs\\job_control-U42CF0cz\\lib\\site-packages\\selene\\elements.py:428: in click\n    condition=be.visible)\n..\\..\\.virtualenvs\\job_control-U42CF0cz\\lib\\site-packages\\selene\\elements.py:251: in _execute_on_webelement\n    return command(_wait_with_screenshot(self._webdriver, self, condition))\n..\\..\\.virtualenvs\\job_control-U42CF0cz\\lib\\site-packages\\selene\\elements.py:193: in _wait_with_screenshot\n    raise TimeoutException(msg, e.screen, e.stacktrace)\nE   selenium.common.exceptions.TimeoutException: Message: \nE               failed while waiting 4 seconds\nE               to assert Visible\nE               for first_by('xpath', '//button[text() = \"Upload\"]')\nE   \nE               reason: NoSuchElementException: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//button[text() = \"Upload\"]\"}\nE     (Session info: chrome=68.0.3440.106)\nE     (Driver info: chromedriver=2.41.578737 (49da6702b16031c40d63e5618de03a32ff6c197e),platform=Windows NT 10.0.16299 x86_64)\nE               screenshot: file://C:\\Users\\ssoloshchenko\\.selene\\screenshots\\1535962116742\\screen_1535962116743.png"}, "start": 1535962124930, "stop": 1535962135796, "uuid": "8f6e7fba-923f-4065-b852-f3334db0e1a8", "historyId": "bd979e64727e67f14073af6141c934aa", "fullName": "tests.test_login#test_upload", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "MOBILE98"}, {"name": "thread", "value": "34988-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}